package.json

```diff
@@ -39,7 +39,7 @@
     "dataloader": "^1.2.0",
     "debug": "^2.2.0",
     "finalhandler": "^0.5.0",
-    "graphql": ">=0.6.0 <1.0.0",
+    "graphql": ">=0.8.0-beta1",
```

Comment:
If it’s so that we can test on `0.8.0`, could we perhaps make `0.8.0` a `devDependency` and keep the wide range in the production dependencies?